src/components/Editable.tsx

```diff
@@ -611,6 +614,7 @@ const Editable = ({ disabled, isEditing, simplePath, path, cursorOffset, showCon
 
     const isHiddenByAutofocus = isElementHiddenByAutoFocus(e.target as HTMLElement)
     const editingOrOnCursor = state.editing || equalPath(path, state.cursor)
+    setIsTapped(true)
```

Comment:
When adding behavior like this that may not be obvious to other developers, it might be good to leave a descriptive comment and link to the issue.